### PR TITLE
Automatically toggle http-to-https redirection with the service

### PR DESCRIPTION
**Pull request by WilliamMcCumstie**
The `flight www <enable|disable>-https` commands previously only turned the `https` server on and off. It did not automatically redirect  `http` requests to the `https` server. In practice the redirection should always be on when the `https` server is enabled. 

There are quite a few services which require `https` for security reasons. These services should only define a `https` config file, which will automatically be picked up when `https` is enabled. A client will probably get `404 Missing` or `502 Bad Gateway` when requesting one of these services when `https` is disabled. Ideally it would be `503 Service Unavailable` but we can live without this.

There maybe services which can operate over `http` as well as `https`. These services should define both a `http` and `https` config file. This way it doesn't matter if `https` is enabled as it will be able to find the service either way.

This PR is based on #41
A few additional commits will be added TBA

### CODE REVIEW

**Comment by WilliamMcCumstie** [comment]
`flight-console-webapp` has been rebuilt without its `http-https` redirect script. Is is available as `0.0.5-3`

**Comment by WilliamMcCumstie** [comment]
`flight-console-api` has been rebuilt without its `http-https` redirect script. It is available as `0.0.2-3`

**Comment by WilliamMcCumstie** [comment]
`flight-desktop-webapp` has been rebuilt without `http-https` redirect script. Is is available as `1.2.0-3`

**Comment by WilliamMcCumstie** [comment]
ATPIT no additional changes are intended. The final packages which need rebuilding will be done on separate branches

File: builders/flight-www/config/software/flight-www.rb [hunk 1]
```diff
@@ -36,15 +36,18 @@
...
+    )
+    if ohai['platform_family'] == 'rhel'
+      conf = tpl.sub('%GROUP%', '')
+    else
+      conf = tpl.sub('%GROUP%', 'nogroup')
+    end
+    File.write(File.join(build_dir, 'nginx.conf'), conf)
   end
-  File.write(File.join(build_dir, 'nginx.conf'), conf)
+
```
Comment (benarmston) [inline builders/flight-www/config/software/flight-www.rb:50]: @WilliamMcCumstie what's the reason for this change?

Comment (benarmston) [reply builders/flight-www/config/software/flight-www.rb:50]: Slack happened at https://alces.slack.com/archives/C5H3J5DCP/p1597396281366000.  This change is probably the correct way to do it.
